src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -132,7 +135,7 @@ public function showDetailsRow($id)
         // get entry ID from Request (makes sure its the last ID for nested resources)
         $id = $this->crud->getCurrentEntryId() ?? $id;
 
-        $this->data['entry'] = $this->crud->getEntry($id);
+        $this->data['entry'] = $this->crud->getEntryWithLocale($id);
```

Comment:
Wait. I don't understand. Why does this PR make changes to the ListOperation? The bug was about changing the UpdateOperation behaviour... not the ListOperation... right?

Reply:
My understanding the bug was in the fact that we didn't set locales on models. (in any operation).
In the update operation it's not enough set the locale on model, so we also provided a way to fill the inputs from other locales. 
In all other operations if you see, I am just setting the locale on model, and created that optional setting I didn't touch anything else in those operations.

But if you feel it don't belong here as I think it does, I can separate those changes into a different PR.